crates/control/src/behavior/dribble.rs

```diff
@@ -39,14 +45,24 @@ pub fn execute(
     };
     let kick_decisions = world_state.kick_decisions.as_ref()?;
     let instant_kick_decisions = world_state.instant_kick_decisions.as_ref()?;
+    let break_precision_kick = break_precision_kick(
+        game_controller_state,
+        precision_kick_timeout,
+        bigger_threshold_start_time,
+        cycle_start_time,
+    );
 
     let available_kick = kick_decisions
         .iter()
         .chain(instant_kick_decisions.iter())
         .find(|decision| {
             is_kick_pose_reached(
                 decision.kick_pose,
-                &in_walk_kicks[decision.variant],
+                if break_precision_kick {
+                    in_walk_kicks[decision.variant].reached_thresholds
+                } else {
+                    in_walk_kicks[decision.variant].precision_kick_reached_thresholds
+                },
```

Comment:
Maybe flip these to indicate that the `precision_kick_reached_thresholds` are the default

Reply:
done